src/main/java/edu/gonzaga/EventCard.java

```diff
@@ -3,6 +3,17 @@
 public class EventCard extends Card{
     private String name;
 
+    /**
+     * Constructor for the EventCard class
+     * @param name the name of the event card
+     * @return an EventCard object
+     * 
+     * @author Tony
+     */
+    public EventCard(String name) {
+        this.name = name;
+    }
+    
     public void playEvent() {
```

Comment:
If we aren't going to use the playEvent() function let's just remove it

Reply:
The event card class is in progress right now, this merge is primarily for the Hand and Deck classes